src/main/java/uk/gov/hmcts/reform/pip/account/management/controllers/MediaApplicationController.java

```diff
@@ -104,7 +105,7 @@ public ResponseEntity<Resource> getImageById(@PathVariable String id) {
     @Operation(summary = "Create a new application")
     @PostMapping(produces = APPLICATION_JSON_VALUE, consumes = MULTIPART_FORM_DATA_VALUE)
     public ResponseEntity<MediaApplication> createApplication(
```

Comment:
Could you add an ApiResponse into the above for the BadRequest response :)

Reply:
Fixed :)